web/next.config.js

```diff
@@ -77,6 +77,9 @@ module.exports = withBundleAnalyzer({
 
     return config;
   },
+  experimental: {
+    largePageDataBytes: 128 * 100000,
+  },
```

Comment:
@jphechter I'd love your input on how to tackle this. This callout component has pushed the page size above 128KB, and while this is a workaround, it's not a good long-term strategy.

Reply:
What?? So we were teetering on the edge up to now? I'm assuming any other change could have done this too?

Reply:
So this is saying we're ok with the page being up to 12.8MB, correct? If we're concerned about page size should we make it 1.28MB as a precaution?